.vscode/settings.json

```diff
@@ -2,5 +2,9 @@
   "[python]": {
     "editor.defaultFormatter": "ms-python.black-formatter",
     "editor.formatOnSave": true
-  }
+  },
+  "cSpell.words": [
```

Comment:
Never seen this, what is it for?

Reply:
Irrelevant. Just added some unknown words to the dictionary.